buildSrc/src/main/java/Dependencies.kt

```diff
@@ -1,179 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-// If you ever need to force a toolchain rebuild (taskcluster) then edit the following comment.
-// FORCE REBUILD 2021-10-06
-
-// Synchronized version numbers for dependencies used by (some) modules
-object Versions {
```

Comment:
You're removing this file by mistake I assume? :)

Reply:
Oh, yes! Thanks for pointing that out.